tests/unit/crs/test_cr_get_vpce_subnets.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+from mock import patch, MagicMock
+
+from backend.lambdas.custom_resources.get_vpce_subnets import (
+    create,
+    delete,
+    handler,
+)
+
+pytestmark = [pytest.mark.unit, pytest.mark.task]
+
+
+@patch("backend.lambdas.custom_resources.get_vpce_subnets.ec2_client")
+def test_it_returns_valid_subnets(mock_client):
+    event = {
+        "ResourceProperties": {
+            "ServiceName": "com.amazonaws.eu-west-2.monitoring",
+            "SubnetIds": ["subnet-0123456789abcdef0", "subnet-0123456789abcdef1"],
+            "VpcEndpointType": "Interface",
+        }
+    }
+
+    mock_client.describe_subnets.return_value = {
+        "Subnets": [
+            {
+                "AvailabilityZone": "eu-west-2a",
+                "SubnetId": "subnet-0123456789abcdef0",
+            },
+            {
+                "AvailabilityZone": "eu-west-2b",
+                "SubnetId": "subnet-0123456789abcdef1",
+            },
+        ]
+    }
+
+    mock_client.describe_vpc_endpoint_services.return_value = {
+        "ServiceDetails": [
+            {
+                "ServiceName": "com.amazonaws.eu-west-2.monitoring",
+                "AvailabilityZones": [
+                    "eu-west-2a",
+                    "eu-west-2b",
+                ],
+            }
+        ]
+    }
+
+    resp = create(event, MagicMock())
+
+    mock_client.describe_subnets.assert_called_with(
+        SubnetIds=["subnet-0123456789abcdef0", "subnet-0123456789abcdef1"]
+    )
+    mock_client.describe_vpc_endpoint_services(
+        Filters=[
+            {
+                "Name": "service-name",
+                "Values": [
+                    "cn.com.amazonaws.u-west-2.monitoring",
+                    "com.amazonaws.eu-west-2.monitoring",
+                ],
+            },
+            {"Name": "service-type", "Values": ["Interface"]},
+        ]
+    )
+
+    assert resp == "subnet-0123456789abcdef0,subnet-0123456789abcdef1"
+
+
+@patch("backend.lambdas.custom_resources.get_vpce_subnets.ec2_client")
+def test_it_raises_exception(mock_client):
+    event = {
+        "ResourceProperties": {
+            "ServiceName": "com.amazonaws.eu-west-2.dummy",
+            "SubnetIds": [],
+            "VpcEndpointType": "Interface",
+        }
+    }
+
+    mock_client.describe_subnets.return_value = {
+        "Subnets": [
+            {
+                "AvailabilityZone": "eu-west-2a",
+                "SubnetId": "subnet-0123456789abcdef0",
+            },
+            {
+                "AvailabilityZone": "eu-west-2b",
+                "SubnetId": "subnet-0123456789abcdef1",
+            },
+        ]
+    }
+
+    mock_client.describe_vpc_endpoint_services.return_value = {"ServiceDetails": []}
+
+    with pytest.raises(Exception) as e_info:
+        create(event, MagicMock())
+
+    mock_client.describe_subnets.assert_called_with(SubnetIds=[])
+    mock_client.describe_vpc_endpoint_services(
```

Comment:
Should this be this instead?

```suggestion
    mock_client.describe_vpc_endpoint_services.assert_called_with(
```